package.json

```diff
@@ -18,6 +18,7 @@
     "precommit": "npm run lint"
   },
   "dependencies": {
+    "@aeternity/aepp-calldata": "github:aeternity/aepp-calldata-js#master",
```

Comment:
we should use the official release here:

`"@aeternity/aepp-calldata": "^1.0.0"`

please also make sure that this is reflected in the package-lock.json after change ;-)